src/MusicStore/test/MusicStore.E2ETests/PublishAndRunTests.cs

```diff
@@ -15,7 +18,7 @@ namespace E2ETests
     public class PublishAndRunTests : LoggedTest
     {
         public static TestMatrix TestVariants
-            => TestMatrix.ForServers(ServerType.IISExpress, ServerType.HttpSys)
+            => TestMatrix.ForServers(ServerType.IISExpress, ServerType.HttpSys, ServerType.Kestrel)
```

Comment:
As you mentioned, I'm never was a fan of the test matrix as it makes it hard to disable things. Maybe we should just scrap it. Not in this PR but eventually.

Reply:
🤷‍♀️